ament_tools/verbs/test/cli.py

```diff
@@ -46,6 +46,10 @@ def prepare_arguments(parser, args):
         default=False,
         help='Stop after tests with errors or failures',
     )
+    parser.add_argument(
+        '-n', '--no-build', dest='build_before_test', default=True, action='store_false',
```

Comment:
I would suggest `--skip-build` since there is also an argument `--skip-install`.

We could also have this argument on the `build` command. When specified it would only perform the install step then.